.github/workflows/issue-opened.yml

```
@@ -28,6 +30,8 @@ jobs:
   set-labels:
     if: ${{ contains(github.event.issue.labels.*.name, 'bug :beetle:') }}
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

```suggestion
    permissions: {}
```
I believe this doesn't need any perms since it's using a GH app token, and it was already inheriting no perms.